### DISCUSSION TITLE
4.7版本，重启consumer，历史一周的消费成功了的消息重新消费了，为什么？

### DISCUSSION THREAD

**Discussion by HeisenbergDong** [apache/rocketmq#7234] [Q&A]
mq是4个broker，2个master，2个salve。 consumer两个节点，其中一个节点网络断线了2个小时，恢复后，重新消费了历史一周的消息。历史被重新消费的消息是之前都成功消费过的消息，为什么呢？求大神指导

**Comment by onejimmyboy** [comment]
消费者设置的setConsumeFromWhere是否合理？
